qutebrowser/commands/runners.py

```diff
@@ -57,6 +84,10 @@ def replace_variables(win_id, arglist):
             QUrl.RemovePassword),
         'clipboard': utils.get_clipboard,
         'primary': lambda: utils.get_clipboard(selection=True),
+        'title': lambda: tabbed_browser.page_title(
+            tabbed_browser.currentIndex()),
+        'domain': lambda: _current_domain(tabbed_browser),
```

Comment:
The whole `_current_domain` function can be stripped down to `url.toString(QUrl::RemovePath)`
(not sure about the python equivalent, see http://doc.qt.io/qt-5/qurl.html#UrlFormattingOption-enum